app/assets/stylesheets/desktop/modal.scss

```diff
@@ -113,19 +113,30 @@
   }
 }
 
-.split-modal {
+.move-to-modal {
   .modal-body {
     position: relative;
     height: 350px;
   }
 
   #move-selected {
+    width: 475px;
+
     p {
       margin-top: 0;
     }
 
-    input[type="radio"] {
-      margin-right: 10px;
+    .radios {
```

Comment:
I noticed in the screenshot that the labels don't line up with the radio buttons. This would be a good place to use flexbox so you could align them properly.